book/reference/IS2-resources.md

```diff
@@ -5,4 +5,3 @@
 Navigating all of the tools available for working with ICESat-2 data can be challenging.
 [NSIDC](https://nsidc.org/data/icesat-2/tools) and [icepyx](https://icepyx.readthedocs.io/en/latest/community/resources.html) have resources pages that provide information on many of the tools available for accessing and processing ICESat-2 data. We've also created this figure to help you navigate some of the tools featured during the hackweek.
 
-![IS2-res-nav](../tutorials/data_access/img/discovery_and_access_methods.png)
```

Comment:
Suggest also removing the line about the figure (end of line 6, above) and making sure we remove the figure itself from the repo.